### PR TITLE
Book schema sql

### LINKED ISSUE #23: [3pt] - DAY 2 Insert and query data in SQL - Member 1
- Create a schema.sql file with tables that will be analogical to the structure of the classes that you created:
   - books table (add all properties and associations from the parent Item class as table columns)
    -  labels table

### PR DESCRIPTION
**Pull request by CarlosE-Feliz**
- Create a schema.sql file with tables that will be analogical to the structure of the classes that you created:
    - books table (add all properties and associations from the parent Item class as table columns)
    - labels table

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,7 +1,12 @@
 /* Database schema analogical to the structure of the classes in our app. */
 
 -- Create book table
```
Comment (Hamzaoutdoors) [inline schema.sql]: Kindly add `books` table and create 1-to-many relationships between your `books` table and [`genres`, `authors`, `labels`] tables 😃 
 **Hint** : You can use foreign keys in your `books` table to create this 1:M relationships

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi, @xJokeer  👋

### Good job so far!🔥🔥
You did a good job implementing the `schema.sql` to create `books` table with all specific properties 🖌️ 

However, some changes need to be implemented before merging into the `dev` branch. Please, take a look at the comments below for more details.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: schema.sql (hunk 1 above)
Comment (CarlosE-Feliz) [reply schema.sql]: Thanks for your feedback!

**Review by Hamzaoutdoors — APPROVED** [review]
## Hi @xJokeer  ,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉  🎉 🎉 

---
Cheers and Happy coding! 👏🏼  👏🏼 👏🏼 

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear_

File: schema.sql (hunk 1 above)
Comment (Hamzaoutdoors) [reply schema.sql]: This issue has been addressed well. Good job! 👏🏼 👏🏼

**Pull request merged**